### DISCUSSION TITLE
Differences between PlayStore- and F-Droid-version and AndroidAuto

### DISCUSSION THREAD

**Discussion by rakor** [PaulWoitaschek/Voice#2478] [Q&A]
Hey there,

I tried to run Voice with AndroidAuto from F-Droid, but the app wasn't shown in the car. In [this hint](https://github.com/PaulWoitaschek/Voice/discussions/1640#discussioncomment-8653140) one said you have to allow untrusted apps, but I found no such setting in AndroidAuto on my Android 13 (running on a Fairphone4).
So I tried the App from the PlayStore (which was shown in AndroidAuto) but was wondering the different version:
PlayStore Version: 8.1.3
F-Droid: 8.2.3

With RethinkDNS I found that the PlayStore-App tries to comminicate with:
- firebaseinstallations.googleapis.com
- scone-psa.googleapis.com
- android.googleapis.com

The F-Droid-App did not try to communicate at all.

I'd like to use the current version with working AndroidAuto and at best without any unnecessary communications.
Can you help?

**Comment by Stephan-4711** [comment]
Can you add the app in the android auto developer settings? There was an option to add "unsupported"  apps
